akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1934,6 +1935,8 @@ trait FlowOpsMat[+Out, +Mat] extends FlowOps[Out, Mat] {
   def watchTermination[Mat2]()(matF: (Mat, Future[Done]) ⇒ Mat2): ReprMat[Out, Mat2] =
     viaMat(GraphStages.terminationWatcher)(matF)
 
+  def monitor: ReprMat[Out, (Mat, FlowMonitor[Out])] = viaMat(Flow.monitor[Out])(Keep.both)
```

Comment:
No, sorry, my bad, this _is_ the right place, I misread the folding the github view and thought it was on `RunnableGraph`. It's a flow that materializes a value - `FlowOpsMat` is right. 

Return wouldn't be a tuple but `ReprMat[Out, FlowMonitor]`, we would also want a `monitorMat` following the pattern of the other methods taking a function that allows you to choose how to handle the materialized value so that it can be called using `Keep.{left, right, both}`